### ISSUE TITLE
[BITV]: focus will be set on actions as far document will be saved

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/text#5326]
The problem is: even if the focus has been set on some NcActions button (like in a sidebar or avatar), after few seconds focus will be moved to one specific NcActions button on the top right. My assuming is that this happens when file will be saved:

![Peek 2024-01-31 15-02](https://github.com/nextcloud/text/assets/6078378/660834da-749d-4676-8bbf-1e263fede5a1)

**Comment by juliushaertl** [comment]
Seems to be an issue also with other viewer file types:

Not sure where this goes wrong then exactly

https://github.com/nextcloud/text/assets/3404133/48503b2e-8455-40bb-b41b-e3db017a0130

**Comment by szaimen** [comment]
the focus trap doesnt seem to work here:

https://github.com/nextcloud/text/assets/42591237/c12a5212-36a8-4915-a377-f1f642bb2704

**Comment by JuliaKirschenheuter** [comment]
> the focus trap doesnt seem to work here:

should be fixed via https://github.com/nextcloud-libraries/nextcloud-vue/pull/4953, waiting for NcVue version `v8.6.0` in `stable28`
